### DISCUSSION TITLE
Read Receipts in chat-messages

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#88] [Ideas & Features]
I believe rather strongly that there should be read receipts so that you can tell if someone has opened and looked at your message. This way you can easily detect if they are ghosting, which would make the reporting of this much more reliable.

**Comment by barvian** [comment]
I agree. I also think if they were part of the free tier (however pricing ends up working), they would be a great differentiator aligned with our values. Tinder, Match, OKC, etc. all offer them as part of their upgrade package, and OKCupid [claims](https://help.okcupid.com/hc/en-us/articles/5221186780045-Read-Receipts) they're "one of the most popular features of our paid memberships".

**Reply by WilliamDormer** [reply]
IMO that is a basic necessity for preventing ghosting. But the apps for sure know this, and thus put it behind a paywall.
